HypVINN/run_prediction.py

```diff
@@ -46,6 +47,8 @@
 from HypVINN.utils.stats_utils import compute_stats
 from HypVINN.utils.visualization_utils import plot_qc_images
 
+_doc_HYPVINN_SEG_NAME = os.path.relpath(HYPVINN_SEG_NAME, FASTSURFER_ROOT)
```

Comment:
I checked, HYPVINN_SEG_NAME is just the file name, no absolute path. so this does not need to be transformed.

Reply:
Removed